include/seqan3/argument_parser/validators.hpp

```diff
@@ -217,26 +254,45 @@ class value_list_validator
 private:
 
     //!\brief Minimum of the range to test.
-    std::vector<value_type> values;
+    std::vector<value_type> values{};
 };
 
 /*!\brief Type deduction guides
  * \relates seqan3::value_list_validator
  * \{
  */
-//!\brief Deduction guide for `std::vector` over an arithmetic type.
-template <arithmetic option_value_type>
-value_list_validator(std::vector<option_value_type>) -> value_list_validator<double>;
+//!\brief Deduction guide for a parameter pack over an arithmetic type.
+template <arithmetic ...option_types>
+value_list_validator(option_types...) -> value_list_validator<double>;
+
+//!\brief Deduction guide for ranges over an arithmetic type.
+template <std::ranges::forward_range range_type>
+//!\cond
+    requires arithmetic<value_type_t<range_type>>
+//!\endcond
+value_list_validator(range_type && rng) -> value_list_validator<double>;
 
-//!\brief Deduction guide for `std::initializer_list` over an arithmetic type.
-template <arithmetic option_value_type>
-value_list_validator(std::initializer_list<option_value_type>) -> value_list_validator<double>;
+//!\brief Deduction guide for a parameter pack over a const char pointer delegates to value type std::string.
```

Comment:
That is only semi-true since all types convertible to std::string are candidates for this deduction overload

Reply:
Good catch, I forgot to adapt the documentation